.github/workflows/tenzir.yaml

```diff
@@ -718,6 +726,11 @@ jobs:
           ccache --show-stats
           # Print statistics about cache compression.
           ccache --show-compression
+      - name: Save ccache Cache
+        uses: actions/cache/save@v3
```

Comment:
Should we add
```
        if: always()
```
here?

Reply:
I'll create a separate PR for that so we can already start building up the ccache now